src/router/driver/libbpf/mempool.c

```diff
@@ -0,0 +1,109 @@
+// +build linux
+
+#include <unistd.h>
+#include <stdint.h>       // for uintptr_t
```

Comment:
is it simpler to use a LIFO queue from [sys/queue.h](http://man7.org/linux/man-pages/man3/queue.3.html)?